en/flight_controller/pixhawk6x-rt.md

```diff
@@ -198,9 +198,11 @@ It is pre-built and automatically installed by _QGroundControl_ when appropriate
 To [build PX4](../dev_setup/building_px4.md) for this target:
 
 ```
-make px4_fmu-v6x_default
+make nxp_fmurt1170-v1_default
```

Comment:
Will this change back when PX4 SWS is enabled?

I think we should also add `git checkout upstream pr-imxrt1170-imxrt1062 # Upstream is PX4-Autopilot repo` above this.

Reply:
Ping

Reply:
the final target name is now px4_fmu-v6xrt

Reply:
correction is done in https://github.com/NXPHoverGames/PX4-user_guide/pull/1

Reply:
```suggestion
make px4_fmu-v6xrt
```